client/src/helpers/skeleton.tsx

```diff
@@ -0,0 +1,23 @@
+import Skeleton from '@mui/material/Skeleton';
+
+export default function CustomSkeleton() {
+  return (
+    <div
```

Comment:
move this to the components folder,  any try to change the style to be more closer to the car card component